workflow/rules/common.smk

```diff
@@ -51,6 +51,67 @@ def get_individual_fastq(wildcards):
     elif wildcards.read == "2":
         return units.loc[ (wildcards.sample, wildcards.unit), "fq2" ]
 
+def get_fastqs(wildcards):
+    """Get raw FASTQ files from unit sheet."""
+    if is_single_end(wildcards.sample, wildcards.unit):
+        return units.loc[ (wildcards.sample, wildcards.unit), "fq1" ]
+    else:
+        u = units.loc[ (wildcards.sample, wildcards.unit), ["fq1", "fq2"] ].dropna()
+        return [ f"{u.fq1}", f"{u.fq2}" ]
+
+def is_control(sample):
+    control = samples.loc[sample]["control"]
+    return pd.isna(control) or pd.isnull(control)
+
+def get_sample_control_combinations():
+    sam_contr = []
+    for sample in samples.index:
+        if not is_control(sample):
+            sam_contr.extend(expand(["{sample}-{control}"], sample = sample, control = samples.loc[sample]["control"]))
+    return sam_contr
```

Comment:
Is this still used anywhere, or just an earlier version of the next function below that remained? If the latter, we could remove this function.